ckanext/xloader/tests/test_plugin.py

```diff
@@ -9,6 +9,26 @@
 from six import text_type as str
 from ckan.tests import helpers, factories
 from ckan.logic import _actions
+from ckan.plugins import toolkit
+from ckanext.xloader.plugin import _should_remove_unsupported_resource_from_datastore
+
+
+@pytest.fixture
+def toolkit_config_value(request):
+    _original_config = toolkit.config.copy()
+    toolkit.config['ckanext.xloader.clean_datastore_tables'] = request.param
+    try:
+        yield
+    finally:
+        toolkit.config.clear()
+        toolkit.config.update(_original_config)
+
+
+@pytest.fixture
+def mock_xloader_formats(request):
```

Comment:
This feels like a convoluted way of applying the flag, but I think it will work.

Reply:
@JVickery-TBS I have raised https://github.com/JVickery-TBS/ckanext-xloader/pull/4 to clean this up